.github/workflows/benchmark_json.yml

```
@@ -0,0 +1,75 @@
+      matrix:
+        package:
+          - biome_json_parser
+          - biome_json_formatter
+          - biome_json_analyze
+
+    steps:
+
+      - name: Checkout PR Branch
+        uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4.3.0
+        with:
+          ref: ${{ steps.sha.outputs.result }}
```

actionlint (1.7.7)

property "sha" is not defined in object type {}

(expression)